src/DIRAC/ResourceStatusSystem/scripts/dirac_rss_set_status.py

```diff
@@ -36,6 +36,8 @@ def registerSwitches():
       ('statusType=', 'StatusType (or comma-separeted list of names), if none applies to all possible statusTypes'),
       ('status=', 'Status to be changed'),
       ('reason=', 'Reason to set the Status'),
+      ('VO=', 'VO to change a status for. Deafault: all '
```

Comment:
```suggestion
      ('VO=', 'VO to change a status for. Default: "all" '
```
But in the other scripts the default used is `None`.